lib/main.js

```diff
@@ -259,7 +259,9 @@ var loadConfig_p = qutil.serialized(function() {
     if (prots.length === 0){
       throw new Error("All protocols were disabled. You must leave at least one protocol enabled.");
     }
-    logger.info("Disabled protocols: " + configRouter.disabledProtocols.join(","));
+    if (configRouter.disabledProtocols && configRouter.disabledProtocols.length > 0){
+      logger.info("Disabled protocols: " + configRouter.disabledProtocols.join(","));
```

Comment:
Debatably. In this case, since the user action is to opt-out, I'm framing the log message to conform to that -- showing them that the protocols are indeed disabled.